music/src/core/visualizer.ts

```diff
@@ -365,13 +361,14 @@ export class PianoRollSVGVisualizer extends BaseVisualizer {
     }
 
     // Remove the current active note, if one exists.
-    const el = this.svg.querySelector('rect.active');
-    if (el) {
+    let el = this.svg.querySelector('rect.active');
```

Comment:
You can just do `let els = this.svg.querySelectorAll('rect.active')`, and just loop through that list instead.

Reply:
Done!